templates/templates/turbo/root/turbo.json.tmpl

```diff
@@ -2,6 +2,7 @@
   "$schema": "https://turbo.build/schema.json",
   "tasks": {
     "build": {
+        "inputs": ["src/**/*", ".fogg-component.yaml"],
```

Comment:
Ensure the `synth` task only runs on changes to src code or fogg.yml components

https://github.com/vincenthsh/fogg/blob/a8ac3b8b4f0523853317674bb92671805d3577c5/templates/templates/turbo/root/package.json.tmpl#L9

Sources:

- https://turborepo.com/docs/crafting-your-repository/configuring-tasks#specifying-inputs
- https://turborepo.com/docs/reference/configuration#globaldependencies
- https://turborepo.com/docs/reference/run#--affected

If this doesn't work, not using `--affected` and implement custom git change detection in the atlantis pre-workflow hook, ultimately calling the `turbo run synth` task with explicit list of packages via `--filter` flags see

- https://turborepo.com/docs/crafting-your-repository/running-tasks#using-filters